src/main/python/pybuilder/utils.py

```diff
@@ -304,3 +305,269 @@ def instrumented_target(*args, **kwargs):
             msg += "\n" + chained_message
         ex = Exception(msg)
         raise_exception(ex, result.args[1])
+
+
+if sys.version_info[0] == 2 and sys.version_info[1] == 6:  # if Python is 2.6
+    # Backport of OrderedDict() class that runs on Python 2.4, 2.5, 2.6, 2.7 and pypy.
+    # Passes Python2.7's test suite and incorporates all the latest updates.
```

Comment:
Actually I'm removing this since the code came from http://code.activestate.com/recipes/576693/ and is available as a package here: https://pypi.python.org/pypi/ordereddict